### PR TITLE
remove unused dependency installation 

### PR DESCRIPTION
**Pull request by wonderflow**
we have reduce 3rd party dependency , they will be installed as capability from registry

### CODE REVIEW

File: pkg/serverlib/capability.go [hunk 1]
```diff
@@ -158,6 +161,25 @@ func InstallCapability(client client.Client, mapper discoverymapper.DiscoveryMap
 	return nil
 }
 
+// HackForStandardTrait will do some hack install for standard capability
+func HackForStandardTrait(tp types.Capability, client client.Client) error {
```
Comment (ryanzhang-oss) [inline pkg/serverlib/capability.go:165]: what about the other traits (autoscale that relies on Keda)?

Comment (wonderflow) [reply pkg/serverlib/capability.go:165]: They don't rely on it. Promethues is special

Comment (zzxwill) [reply pkg/serverlib/capability.go:165]: Autoscale replies on KEDA.

Comment (wonderflow) [reply pkg/serverlib/capability.go:165]: let me explain more clear:

autoscale relies on KEDA, it's installed by helm in the definition file https://github.com/oam-dev/catalog/blob/master/registry/autoscale.yaml#L17 The helm chart is running the keda instance.

Promethues is special, when we install the helm chart, we just installed an operator which will be able to create promethus instance, it needs an extra CR to create an instance for runing promethus instance. 

They are in different level.

**Review by zzxwill — APPROVED** [review]
LGTM

**Pull request merged**